login/daint-gpu/21.09

```diff
@@ -33,23 +31,24 @@ if { [module-info mode remove] || [module-info mode switch] } {
  foreach module ${toolsgpulist} {
    module unload ${module}
  }
-# unload cdt and cdt-cuda
- module unload cdt cdt-cuda
 }
 
 # load dependency and update MODULEPATH
 module load craype-haswell
 module use "${gpupath}"
 module use "${toolsgpupath}"
 
+# load additional modules
+#module load xalt
```

Comment:
@victorusu We are currently not loading `xalt`, should we load it by default?

Reply:
yes, we should

Reply:
Wait... @lucamar, @gppezzi and @vkarak, I am not sure... xalt is loaded by default from the system point of view... Here it is about "loading" it when `module load daint-{gpu,mc}`... I am not sure about this...
Because if the user is unloading it already, then it is a bit annoying to unload it another time, just because it has loaded `daint-{gpu,mc}`, no?
I think we should leave it as it is ... What do you think?

Reply:
That's fine with me, thanks for clarifying @victorusu